docs/python/reference/table-operations/select/view.md

```diff
@@ -19,6 +19,9 @@ When using `view`, the data being requested is not stored in memory. Rather, a f
 > [!CAUTION]
 > When using `view` or [`update_view`](./update-view.md), non-deterministic methods (e.g., random numbers, current time, or mutable structures) produce _unstable_ results. Downstream operations on these results produce _undefined_ behavior. Non-deterministic methods should use [`select`](./select.md) or [`update`](./update.md) instead.
 
+> [!CAUTION]
```

Comment:
Should the link also appear at the bottom of the document?  I could go either way.  Should there also be a link to the parallelization quick start?